### DISCUSSION TITLE
Composite primary keys on web UI

### DISCUSSION THREAD

**Discussion by AndrewHows** [supabase/supabase#1658] [Questions] [answered]
Is there any way to create a table using a composite primary key (e.g. M2M join table with the key being each of the foreign keys)?

I can't find any way to do it via the web interface, and the user in the psql connection string doesn't appear to be able to modify the schema (postgres instead of supabase_admin)

**Comment by soedirgo** [comment, marked as answer]
Hey @AndrewHows, you can use the SQL editor (which has the role of `supabase_admin` by default) to add the composite primary key, though the key isn't exposed in the table editor at the moment.

**Reply by AndrewHows** [reply]
Ah, right. Took me a while to find it - I was looking inside the DB tab, rather than expecting it to have a tab of its own.
